src/python/pull_gsheets_translations_mvp.py

```diff
@@ -0,0 +1,299 @@
+	"""
+	# if value is a string with more than one word
+	if len(value.split()) > 1:
+		#converts value to camelCase
+		camelCase = value.split()[0].lower() + " ".join(value.split()[1:]).title().replace(" ","")
+		#removes punctuation
+		return camelCase.translate(str.maketrans('', '', punctuation))
+	else:
+		return value.lower().translate(str.maketrans('', '', punctuation))
+
+def depunctuate(text):
+	""" Removes punctuation from text.
+
+	Arguments:
+		text {[string]} -- [any string]
+
+	Returns:
+		[string] -- [string with punctuation removed]
+	"""
+	chars = punctuation
+	for c in chars:
+		text = text.replace(c, "")
+	return text
+
+def education_value_cleaner(language_df):
+	""" Converts the long text in the Education 'value' column to shorter text to be used as JSON keys.
+
+	Arguments:
+		language_df {[Series]} -- [A pandas series containing the 'value' column from the 'Education' sheet ]
+
+	Returns:
+		[Series] -- [A pandas Series with shortened text and no punctuation.]
+	"""
+	lst = [word_tokenize(x) for x in language_df.value.tolist()]
+	#http://www.nltk.org/book_1ed/ch05.html
+	pos_tagged = [pos_tag(x) for x in lst]
+	lst_new=[]
+	for pt in pos_tagged:
+		#choose nouns in sentence
+		nouns = [depunctuate(x[0]) for x in pt if "NN" in x[1]]
+		#choose verbs in setence
+		verbs = [depunctuate(x[0]) for x in pt if 'VB' in x[1]]
+		#make fieldKey and add to new list of fieldKeys
+		if len(verbs)==0 and len(nouns)==2:
+			lst_new.append(nouns[0].lower() + nouns[1].title())
+		elif len(verbs)==1 and len(nouns)==1:
+			lst_new.append(nouns[0].lower() + verbs[0].title())
+		else:
+			lst_new.append(nouns[0].lower() + nouns[1].title() + verbs[0].title())
+	return Series(lst_new,index=language_df.index)
+
+def survey_value_cleaner(language_df):
+	""" Converts the long text found within the 'Survey' sheets to shorter text to be used as JSON keys.
+
+	Arguments:
+		language_df {[DataFrame]} -- [A pandas DataFrame containing the information from the 'Survey' sheet.]
+
+	Returns:
+		[DataFrame] -- [Returns a new DataFrame with shortended text and no punctuation.]
+	"""
+	vPA_col = [i for i,x in enumerate(language_df.columns) if x=='valuePossibleAnswers'][0]
+	tVPA_col = [i for i,x in enumerate(language_df.columns) if x=='translatedValuePossibleAnswers'][0]
+	v_col = [i for i,x in enumerate(language_df.columns) if x=='value'][0]
+	tV_col = [i for i,x in enumerate(language_df.columns) if x=='translatedValue'][0]
+	rows = language_df.values.tolist()
+	rows_added = []
+	for i,row in enumerate(rows):
+		orig_row = row.copy()
+		orig_row[v_col] = convert_to_camelCase(orig_row[v_col])
+		rows_added.append(orig_row)
+		arr = row[vPA_col].split(';')
+		tarr = row[tVPA_col].split(';')
+		if len(arr)==len(tarr) and len(arr)>3:
+			for j,_ in enumerate(arr):
+				row_new = row.copy()
+				row_new[v_col] = arr[j].replace('\n','')
+				row_new[v_col] = convert_to_camelCase(row_new[v_col]).replace(" ","")

```

Comment:
```suggestion
				row_new[v_col] = convert_to_camelCase(row_new[v_col])
```

`convert_to_camelCase` does the `replace` I believe